docs/tutorials/01_ui/README.rst

```diff
@@ -0,0 +1,4 @@
+Basic User Interface Element
+----------------------------
+
+These Examples show how to create basic user interfaces element.
```

Comment:
Examples -> tutorials (to be consistent with the rest of the docs)
basic user interfaces element. -> basic user interface elements.